src/components/AboutDialog.tsx

```diff
@@ -61,6 +56,7 @@ const AboutDialog = ({
     .join("\n");
 
   const { hasCopied, onCopy } = useClipboard(clipboardVersion);
+  const { OrgLogo } = useDeployment();
```

Comment:
Makes sense to destructure OrgLogo on line 45 with appNameFull?

Reply:
9637ca2cce31bf7286d7f1b01f62555956514b81